llvm/test/CodeGen/NVPTX/alias.ll

```diff
@@ -24,6 +30,10 @@ attributes #0 = { noreturn }
 ;      CHECK: .visible .func noreturn()
 ; CHECK-NEXT: .noreturn
 
+;      CHECK: .visible .func  (.param .b32 func_retval0) z()
+;      CHECK:      call.uni (retval0), 
+; CHECK-NEXT:      b,
```

Comment:
It would be good to add a RUN line for checking the generated code w/ ptxas.

Reply:
Good suggestion. As it turns out this exposed more issues with the existing implementation. I've update the description with more info.